### ISSUE TITLE
Consumers of the same group consuming same partitions and getting same messages

### ISSUE THREAD

**Issue by juliorenner** [kafka-rust/kafka-rust#211]
I am using the kafka library and have multiple replicas of my consumers to ensure HA. However, I have noticed that even that I specify the consumer group and the client id, that the replicas are getting the same messages from kafka.

After logging the consumer partitions, I noticed that they are the same.

Therefore, I just want to confirm if it is possible to have multiple consumers for the same topic, consuming messages from different partitions without I manually specifying which partition should be consumed. My goal is to avoid the same messages to be processed multiple times by different consumers.

**Comment by johnward** [comment]
Yes it it, you just need to manage the offset accordingly. Consumers in Kafka are very light touch. thanks

**Comment by mrcolo** [comment]
Can you be a bit more verbose on how to solve this problem? Perhaps provide some example code of how one would go about managing the offsets accordingly?
